src/vsr/replica.zig

```diff
@@ -3026,17 +2692,26 @@ pub fn ReplicaType(
         ///
         /// The caller owns the returned message, if any, which has exactly 1 reference.
         fn create_view_change_message(self: *Self, command: Command) *Message {
```

Comment:
We create the message, but then only send it if the `view` matches the `view_durable` (and in the case of DVC, if the `log_view` matches the `log_view_durable`. (See `send_message_to_replica()`).

There is a bit of subtlety here, though: We synchronously send the message to _ourself_ (when we are the primary), even if the view/log_view is not durable yet. This simplifies the `on_do_view_change()`, which expects to always see a DVC from itself.